tests/MarginCalculatorIntegrationTest.php

```diff
@@ -0,0 +1,272 @@
+<?php
+
+namespace PivotLibre\Tideman;
+
+use PivotLibre\Tideman\TestScenario1;
+use PivotLibre\Tideman\MarginCalculator;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\EventDispatcher\GenericEvent;
+
+/**
+ * Each of these large tests were constructed as follows:
+ *
+ * Visit the Google Sheet:
+ * https://docs.google.com/spreadsheets/d/1634wP6-N8GG2Fig-yjIOk7vPBn4AijXOrjq6Z2T1K8M/edit?usp=sharing
+ *
+ * (Optionally create a worksheet describing a collection of ballots that are grouped with a count)
+ * Create a worksheet describing a collection of individuals' ballots.
+ * Create a worksheet describing the expected margins.
+ *
+ * Download the individual ballots worksheet as a TSV.
+ * Generate a php class from the TSV by running:
+ *     python parse_ballot.py <name_of_individual_ballot.tsv> > TestScenarioN.php
+ * Update the name of the generated class to match the name of the file it is inside of.
+ * Create a new method in MarginCalculatorIntegrationTest.
+ * Inside of the method, manually construct a list of expected margins by looking at the expected margins worksheet.
+ * Compare the expected margins and the actual margins using the `checkMargins()` utility method.
+ *
+ */
+class MarginCalculatorIntegrationTest extends TestCase
+{
+    protected function checkMargins($expectedMargins, $ballots)
+    {
+        $agenda = new Agenda(...$ballots);
+        $marginRegistry = (new MarginCalculator())->calculate($agenda, ...$ballots);
+        $this->assertEquals(sizeof($expectedMargins), $marginRegistry->getCount());
+        foreach ($expectedMargins as $expectedMargin) {
+            $winner = $expectedMargin->getWinner();
+            $loser = $expectedMargin->getLoser();
+            $marginAsString = "{$winner->getId()} -> {$loser->getId()}";
+            $actualMargin = $marginRegistry->get($winner, $loser);
+            $this->assertEquals($expectedMargin, $actualMargin, $marginAsString);
+        }
+    }
+
+    public function testScenario1() : void
```

Comment:
I asked Lucas about this. No response yet.

Reply:
Resolved. Lucas independently confirmed my suspicion -  that the original margin values in the Scenario 1 worksheet were double what they should have been. He found the same problem in the Scenario 5 worksheet, which is not part of this pull request. The margin values for Scenario 1 in this pull request are half of the original values in the worksheet, and the tests pass.  w00t!